### ISSUE TITLE
[BUG] Secondary IP address does not get when USB to ETHERNET device is used

### ISSUE THREAD

**Issue by slinkfurball** [RunOnFlux/fluxnode-multitool#19]
### Environment

Self-Hosted

### System

20.04 LTS

### Version

v7.2

### Describe the problem

When using USB to Ethernet dongle to install the fluxnode (option#2)in  multitoolbox, the secondary IP address does not get created for the ethernet interface.    I have tried both TP-LINK Model UE-300 USB 3.0 to Gigabit Ethernet Adapter and Belkin USB 2.0 Ethernet Adapter F4U047 100Mbps and both have the same issue.    

Users will see "Failed: Could not get Node Tier From Flux OS"  in the ~/.fluxbenchmark/debug.log and option #3 does not provide helpful insight as to what the problem maybe.   Improvement in option #3 is to report if secondary IP is missing and how to fix it.

Temporary Work around is to add the secondary interface manually at boot up via the following commands:

sudo ip addr add <public IP / Netmask> dev <interface>

run the following cmd to see the details of the interface:  ip addr
priminary IP:  LAN IP/netmask
secondary IP : Public IP/netmask  <-- this one is missing after multitoolbox install of fluxnode (option #2)

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using the latest version of multitoolbox
- [X] You've checked that this [issue hasn't already been raised](https://github.com/RunOnFlux/fluxnode-multitool/issues?q=is%3Aissue)
- [X] You've checked the official [RunOnFlux Discord](https://discord.io/runonflux) Pinned messages and FAQ ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/RunOnFlux/fluxnode-multitool/blob/master/.github/CODE_OF_CONDUCT.md)

**Comment by floriangrousset** [comment]
Multiple scripts [multitoolbox.sh](https://github.com/RunOnFlux/fluxnode-multitool/blob/91a6c8ab00f47eef4b0c5bc7714ca2dfdd020b9c/multitoolbox.sh), [nodeanalizerandfixer.sh](https://github.com/RunOnFlux/fluxnode-multitool/blob/91a6c8ab00f47eef4b0c5bc7714ca2dfdd020b9c/nodeanalizerandfixer.sh) and [install_pro.sh](https://github.com/RunOnFlux/fluxnode-multitool/blob/91a6c8ab00f47eef4b0c5bc7714ca2dfdd020b9c/install_pro.sh) try to locate the physical ethernet adapter on the host using `BROADCAST,MULTICAST,UP,LOWER_UP` to add the public IP to it. It could pose a problem if using certain USB adapters returning a different string. Exact same problem if you want to self-host your node behind a VPN where `ip addr` would for example return `POINTOPOINT,NOARP,UP,LOWER_UP` on a wireguard interface.
Are there any plans for adjusting this in the close future?
@matthewjdegarmo @XK4MiLX

**Comment by XK4MiLX** [comment]
i will go in diff direction due not all user are networking expert and solution should work for all
can you dm me on discord  :)

**Comment by floriangrousset** [comment]
Agree, that could add a layer of difficulty for less tech-savvy node operators so better keep it simple as default behavior. What's your handle on the Flux discord? I've looked up for XK4MiLX but can't seem to find you.

**Comment by XK4MiLX** [comment]
New solution in development branch => option 10 > Device - create config
How run script form development branch:
```multitoolbox development```

**Comment by TechDufus** [comment]
@floriangrousset Are you still having issues with this?
